draft-ietf-mls-protocol.md

```diff
@@ -4954,33 +4993,29 @@ To construct the tree in {{parent-hash-tree}}:
 
 # Array-Based Trees
 
-One benefit of using left-balanced trees is that they admit a simple
```

Comment:
I would've leaned towards leaving this section as it was, since left-balanced and complete are supposed to be functionally equivalent but left-balanced is more memory efficient.